src/components/Navigation/styles.scss

```
@@ -0,0 +1,88 @@
+// Mobile portrait
+.wrapper {
+	width: 100%;
+	height: 100%;
+	padding-top: 10vh;
+	background-color: red;
+}
+
+	.itemContainer {
```

```suggestion
	.container {
```
no conflict with the nav button anymore 🙌